.travis.yml

```diff
@@ -18,22 +18,24 @@ services:
 
 matrix:
   include:
-    - php: 5.3
+    - php: 5.4
       env: COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
hmm .. decided to revert removing the hack .. but now ``--prefer-lowest`` is passing and everything else is failing .. but this looks fixable .. will try to figure it out over the weekend. but if someone has some time to kill I wouldn't mind a PR :)